### PR TITLE
Feature/add profile picture

### LINKED ISSUE #14: As a user, I should be able to upload my profile photo from device storage.

### PR DESCRIPTION
**Pull request by tylrs**
## Description

-  Add view and tests for editing/uploading a profile picture
-  Add ability to see profile picture displayed on preview pane and in sun
-  Update fetch URL to reflect backend changes

## Concerns

-  Any bugs still present?
-  Next steps? still need to post/patch profile picture to back end 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

closes #14

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -11,15 +11,33 @@ import familyTree from "../../assets/family-tree.png";
...
-  const [user, setUser] = useState({})
+  const [user, setUser] = useState({
+      "id": "",
+      "type": "",
+      "attributes": {
+          "email": "",
+          "name": "",
+          "date_of_birth": "",
+          "obituary": "",
+          "profile_picture": "",
```
Comment (AshleyOh-bit) [inline src/components/App/App.js:27]: You should change this to be the correct name that will be coming back from the fetch.

File: src/components/PhotoAdd/PhotoAdd.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import './PhotoAdd.css'
+
+const PhotoAdd = ({updateProfilePicture, currentProfilePic}) => {
+    const [photoFilePath, setPhotoFilePath] = useState(currentProfilePic)
+    const [previewHeader, setPreviewHeader] = useState("Current Profile Picture")
+
+    const handleSubmit = (e) => {
+        setPreviewHeader("Current Profile Picture")
```
Comment (AshleyOh-bit) [inline src/components/PhotoAdd/PhotoAdd.js:9]: We may have to add a parameter to this function to make this component reusable.

**Review by AshleyOh-bit — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (tylrs) [reply src/components/App/App.js]: Removed this!

File: src/components/PhotoAdd/PhotoAdd.js (hunk 2 above)
Comment (tylrs) [reply src/components/PhotoAdd/PhotoAdd.js:9]: Yep we will have to change quite a few things with the new condition

**Pull request merged**